src/ui/public/timepicker/timepicker.js

```diff
@@ -24,7 +24,9 @@ module.directive('kbnTimepicker', function (quickRanges, timeUnits, refreshInter
       to: '=',
       mode: '=',
       interval: '=',
-      activeTab: '='
+      activeTab: '=',
+      onFilterSelect: '=',
```

Comment:
@weltenwort It doesn't look like the `from` and `to` arguments are being passed anyway? But to pass them, Angular wants you to do in this _super intuitive way_ (sarcasm :)

```html
on-filter=select="updateFilter(from, to)"
```

```javascript
onFilterSelect({ from, to });
```

@Bargs I couldn't think of any disadvantages either because it was always just "the way to do things". So I did some digging and the only thing I could find are some vague references on StackOverflow about how the expression interpreted by `&` is executed in the parent scope, but when interpreted by `=` it's executed in the call site's scope. The actual Angular docs don't really make this clear.

So I make this [pen](http://codepen.io/cjcenizal/pen/WoBvQW) as a test. Two directives, each one representing each approach. They each receive a callback and call it immediately, which logs out `this`. If you open up console you can see that `this` evaluates to the object in which the callback is defined when using `&` and the directive that calls the callback when using `=`.

So I think the only real advantage is that `&` will bind the callback to the right scope.